include/nuttx/input/kbd_codec.h

```diff
@@ -223,7 +223,7 @@ extern "C"
  *
  ****************************************************************************/
 
-#define kbd_press(ch, stream) (stream)->put((stream), (int)(ch))
+#define kbd_press(ch, stream) lib_stream_putc(stream, ch)
```

Comment:
maybe now can use `lib_stream_putc` directly?

Reply:
it's better to keep kbd_press, since there are more complex routines(e.g. kbd_specpress) which map one key event to several character.

Reply:
Sure